src/CSET/recipes/aggregate_precipitation.yaml

```diff
@@ -0,0 +1,32 @@
+title: Extract sub-hourly or hourly ppn and aggregate to user defined intervals
+description: |
+  Extracts out the sub-hourly or hourly stratiform rainfall rate from a file
+  and aggregates it up to new intervals and writes it to a new one.
+section: Grid Stat
+major_cat: Major Category 1
+minor_cat: Minor Category 1
+
+steps:
+  - operator: read.read_cubes
+
+  - operator: filters.filter_cubes
+    constraint:
+      operator: constraints.combine_constraints
+      stash_constraint:
+        operator: constraints.generate_stash_constraint
+        stash: m01s04i203
```

Comment:
Might want to add a comment explaining this stash code.

Reply:
Added stash code translation into description section